pydax/_high_level.py

```diff
@@ -64,7 +65,7 @@ def init(update_only: bool = True, **kwargs: Any) -> None:
     :param LICENSE_SCHEMA_URL: The default license schema file URL.
     :param DATADIR: Default dataset directory to download/load to/from. The path can be either absolute or relative to
         the current working directory, but will be converted to the absolute path immediately in this function.
-        Defaults to: ~/.pydax/data
+        Defaults to: :file:`~/.pydax/data`.
```

Comment:
Another issue is what this means for Windows users. We can fix later if you what to.

Reply:
yeah agree we can fix later